docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -0,0 +1,756 @@
+# End-to-End Guide: Headless Checkout Flow with GraphQL APIs
+
+BigCommerce provides GraphQL Storefront APIs used by server-side and browser-side clients for managing cart and checkout objects to build storefront features. The GraphQL APIs offer the same use cases as our Cart and Checkout REST Management APIs but have the added benefit of making it easier to build headless storefront applications. Developers using GraphQL can spend less time focusing on API interactions and more time building the shopper experience.
+
+This end-to-end guide demonstrates how to power headless storefronts and other non-Stencil shopper experiences. These steps will allow you to build shopping experiences where the purchase funnel is entirely powered by GraphQL, from catalog browsing through checkout. The examples walk you through critical GraphQL capabilities to fit most use cases when creating a headless storefront:
```

Comment:
I'm not sure there's are "other non-Stencil shopper experiences". I think "headless storefront" describes any such storefront.